.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
+            fftw-arch: 'x32'
+            cmake-arch: 'Win32'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win32_msvc2019'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll32.zip'
+            cmake-generator: 'Visual Studio 16 2019'
+            msvc-year: '2019'
+            vcvars-script: 'vcvars32.bat'
+            chocolatey-options: '--forcex86' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'ON' # might need to be turned off for MinGW
+            ableton-link: 'ON' # might need to be turned off for MinGW
+            artifact-suffix: 'win32' # set if needed - will trigger artifact upload
+            create-installer: ${{ startsWith(github.ref, 'refs/tags/') }}
+            installer-suffix: 'win32-installer'
+
+          - job-name: '64-bit'
+            fftw-arch: 'x64'
+            cmake-arch: 'x64'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win64_msvc2019_64'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll64.zip'
+            cmake-generator: 'Visual Studio 16 2019'
+            msvc-year: '2019'
+            vcvars-script: 'vcvars64.bat'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'ON' # might need to be turned off for MinGW
+            ableton-link: 'ON' # might need to be turned off for MinGW
+            artifact-suffix: 'win64' # set if needed - will trigger artifact upload
+            create-installer: ${{ startsWith(github.ref, 'refs/tags/') }}
+            installer-suffix: 'win64-installer'
+
+          - job-name: '64-bit MinGW'
+            fftw-arch: 'x64'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win64_mingw81'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll64.zip'
+            cmake-generator: 'MinGW Makefiles'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'OFF' # might need to be turned off for MinGW
+            ableton-link: 'OFF' # might need to be turned off for MinGW
+            artifact-suffix: 'win64-mingw' # set if needed - will trigger artifact upload
+
+    name: Windows ${{ matrix.job-name }}
+    env:
+      BUILD_PATH: ${{ github.workspace }}/build
+      INSTALL_PATH: ${{ github.workspace }}/build/Install
+      LIBS_DOWNLOAD_PATH: ${{ github.workspace }}/../3rd-party
+      ASIO_URL: 'http://www.steinberg.net/sdk_downloads/asiosdk2.3.zip'
+      ASIO_FOLDER_NAME: ASIOSDK2.3
+      VCVARS_SCRIPT: ${{ matrix.vcvars-script }}
+      VCVARS_SCRIPT_PATH: 'C:/Program Files (x86)/Microsoft Visual Studio/${{ matrix.msvc-year }}/Enterprise/VC/Auxiliary/Build/${{ matrix.vcvars-script }}'
+      ARTIFACT_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.artifact-suffix }}'
+      INSTALLER_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.installer-suffix }}'
```

Comment:
seems you can eliminate this, too

Reply:
This one I'd like to keep in env, artifact names are a little bit fragile element of the workflow since they need to match between jobs, so I'd prefer not to bury it inside one of the steps